chore(server-events): adding integration documentation (#843)
front-commerce/developers.front-commerce.com: docs/02-guides/server-side-events/faq.mdx

---
title: FAQ
description:
  Small questions, quick answers about Server events
sidebar_position: 3
---

<p>{frontMatter.description}</p>

> ## How can i test Server events in local environment?

Reviewer: ```suggestion
## How can I test Server events in local environment?
```

> To test server events in local environment you need a Redis server, and setup it's IP in your `.env` file :

Reviewer: ```suggestion
To test server events in a local environment, you need a Redis server, and you must configure the IP address in your .env file:
```

```
FRONT_COMMERCE_SERVER_EVENT_REDIS_HOST=127.0.0.1
# The following entry is to be defined only if you don't use default Redis port
FRONT_COMMERCE_SERVER_EVENT_REDIS_PORT=6379
```

> Then you need to start the worker by running this command from your project : `pnpm run worker`

Reviewer: ```suggestion
Then you need to start the worker by running the following command from your project : `pnpm run worker`
```

> If you want to have some more infos and activate debug mode, you can either :

Reviewer: ```suggestion
If you would like to obtain additional information and enable debug mode, you can do one of the following:
```

- Run the command with the DEBUG flag : `DEBUG=front-commerce:server-events:cli pnpm run worker`
- Add the debug flag to your `.env` file : `DEBUG=front-commerce:server-events:cli`

> ## What are the data returned by the Server events?

Reviewer: ```suggestion
## What data is published for a Server Event?
```
